airflow/utils/db.py

```diff
@@ -286,6 +284,16 @@ def initdb(rbac=False):
             conn_id='cassandra_default', conn_type='cassandra',
             host='cassandra', port=9042))
 
+
+def initdb(rbac=False):
+    session = settings.Session()
+
+    from airflow import models
```

Comment:
is there a reason we put `airflow import models` as local import not global import?

Reply:
@feng-tao It is to prevent a circular import